deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: upgradeconfigs.upgrade.managed.openshift.io
+spec:
+  group: upgrade.managed.openshift.io
+  names:
+    kind: UpgradeConfig
+    listKind: UpgradeConfigList
+    plural: upgradeconfigs
+    shortNames:
+    - upgrade
+    singular: upgradeconfig
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.phase
+      name: status
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: UpgradeConfig is the Schema for the upgradeconfigs API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: UpgradeConfigSpec defines the desired state of UpgradeConfig
+              and upgrade window and freeze window
+            properties:
+              desired:
+                description: Specify the desired release
+                properties:
+                  channel:
+                    description: Channel we gonna use for upgrades
+                    type: string
+                  force:
+                    default: false
```

Comment:
Is the default value redundant if it is required?

Reply:
I don't think so